### ISSUE TITLE
ConvolutionFilter creates lighten border

### ISSUE THREAD

**Issue by wyllisMonteiro** [pixijs/pixi-filters#212]
/*!
 * pixi.js - v4.8.1
 * Compiled Wed, 06 Jun 2018 15:38:05 UTC
 *
 * pixi.js is licensed under the MIT License.
 * http://www.opensource.org/licenses/mit-license
 */

Hello eeryone, I'm using convolutionFilter() to create sharpen effect with matrix and I got border
Look at the following image

![Capture](https://user-images.githubusercontent.com/36091415/58705720-5381a980-83b0-11e9-9111-bd72d8493ef5.PNG)

**Comment by bigtimebuddy** [comment]
Yeah, this seems like a bug: https://jsfiddle.net/bigtimebuddy/bqy2m1zk/
Weird border issues.
